pkg/security/module/rate_limiter.go

```diff
@@ -47,31 +48,46 @@ func NewLimiter(limit rate.Limit, burst int) *Limiter {
 
 // RateLimiter describes a set of rule rate limiters
 type RateLimiter struct {
-	limiters map[string]*Limiter
+	sync.RWMutex
```

Comment:
We discussed that it might not be that impactful, but we could get rid of this mutex by using the same array of `map[rules.RuleID]*Limiter` + swap method.